static/js/components/email/EmailCompositionDialog.js

```diff
@@ -84,7 +84,7 @@ export default class EmailCompositionDialog extends React.Component {
     this.state = editorStateFromProps(props)
   }
 
-  componentWillReceiveProps(nextProps: EmailDialogProps) {
+  getDerivedStateFromProps(nextProps: EmailDialogProps) {
```

Comment:
Same comment as above. Does this need to be a static method?
![Screenshot from 2019-09-10 17-22-11](https://user-images.githubusercontent.com/863262/64652503-08852280-d3f2-11e9-827a-70bd8f406c41.png)